lib/src/utils.dart

```diff
@@ -104,3 +105,84 @@ String getResponseError(Response res) {
 
 Future<String> ownerToAddress(String owner) async => encodeBytesToBase64(
     await sha256.hash(decodeBase64ToBytes(owner)).then((res) => res.bytes));
+
+Uint8List longTo32ByteArray(int long) {
```

Comment:
Tech debt ticket for unit tests

Reply:
Same for all below.

Reply:
[Created tech debt ticket ](https://ardrive.atlassian.net/jira/software/c/projects/PE/boards/2?modal=detail&selectedIssue=PE-595)